addons/addon-environment-sc-api/packages/environment-type-mgmt-services/lib/environment-type/env-type-config-service.js

```diff
@@ -241,6 +240,33 @@ class EnvTypeConfigService extends Service {
     return savedConfig;
   }
 
+  async checkAndUpdateParams(envType, config) {
```

Comment:
Setting default values for the env params, depending on if AppStream is turned on or not.